nixos/modules/services/logging/filebeat.nix

```diff
@@ -41,23 +47,28 @@ in
 
           See <https://www.elastic.co/guide/en/beats/filebeat/current/configuration-filebeat-options.html>.
         '';
-        default = {};
-        type = types.attrsOf (types.submodule ({ name, ... }: {
-          freeformType = json.type;
-          options = {
-            type = mkOption {
-              type = types.str;
-              default = name;
-              description = ''
-                The input type.
+        default = { };
+        type = types.attrsOf (
```

Comment:
I don't think the module system will now properly merge this.

I think this could be similar to https://github.com/NixOS/nixpkgs/commit/490ba58dd7a5687a5cde953161a5bcb198261012 but not fully sure.

Reply:
Looks like it is working. I tested a couple configurations and just had one issue with an assertion. Also this is nothing I changed in this commit, it was just changed due to new formatting standard which has been introduced in https://github.com/NixOS/nixpkgs/pull/326407.